catalog/templates/catalog/contacts.html

```diff
@@ -0,0 +1,156 @@
+                   aria-expanded="false">
+                    Пользователь
+                </a>
+                <ul class="dropdown-menu ">
+                    <li><a class="dropdown-item" href="#">Профиль</a></li>
+                    <li>
+                        <hr class="dropdown-divider">
+                    </li>
+                    <li><a class="dropdown-item" href="#">Выход</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="col-10">
+
+
+            <div class="pricing-header px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center">
+                <h1 class="display-4">Контакты</h1>
+            </div>
+
+            <div class="container">
+                <div class="row text-start">
+                    <div class="col-lg-9 col-md-6 col-sm-12">
+                        <div class="card mb-4 box-shadow">
+                            <div class="card-body">
+                                <table class="table table-striped">
+                                    <tr>
+                                        <td class="w-25">Страна</td>
+                                        <td>Россия</td>
+                                    </tr>
+                                    <tr>
+                                        <td class="w-25">ИНН</td>
+                                        <td>421719428111</td>
+                                    </tr>
+                                    <tr>
+                                        <td class="w-25">Адрес</td>
+                                        <td>Новокузнецк, Октябрьский пр-т 3-47</td>
+                                    </tr>
+                                </table>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-lg-3 col-md-6 col-sm-12">
+                        <div class="card mb-4 box-shadow">
+                            <div class="card-header">
+                                <h4 class="my-0 font-weight-normal">Свяжитесь с нами</h4>
+                            </div>
+                            <div class="card-body">
+                                <form method="post" action="" class="form-floating">
+                                    {% csrf_token %}
+                                    <div class="mb-3">
+                                        <label for="name">Имя</label>
+                                        <input type="text" name="name" class="form-control" id="name"
+                                               placeholder="Ваше имя"
+                                               required>
+                                    </div>
+                                    <div class="mb-3">
+                                        <label for="phone">Телефон</label>
+                                        <input type="text" name="phone" class="form-control" id="phone"
```

Comment:
Больше подходит `tel`. Можешь почитать и о других типах https://htmlbook.ru/html/input/type

```suggestion
                                        <input type="tel" name="phone" class="form-control" id="phone"
```